Console.Extensions.Blazor/BlazorConsole.cs

```diff
@@ -146,8 +168,21 @@ public Task WriteAsync(string consoleInput)
         protected IJSRuntime? JSRuntime { get; set; }
         private bool isFirstUse = true;
         protected string ConsoleInputId = "consoleInput";
+        /// <summary>
+        /// Generates the proper output for Console.Write or Console.WriteLine for the web browser
+        /// </summary>
+        /// <remarks>
+        /// The generated output will be written to <see cref="Output"/>. 
+        /// </remarks>
+        /// <param name="consoleInput"></param>
+        /// <param name="newline"></param>
+        /// <returns></returns>
         private async Task WriteLinePrivate(string consoleInput, bool newline = true)
         {
+            if (ReplaceWhiteString)
```

Comment:
@TrevorDArcyEvans I had to move the replacement of " " to the top, otherwise it would also replace things like `<span class=\"Gray\">` to `<span&nbsp;class=\"Gray\">`